docs/source/tutorial.rst

```diff
@@ -117,56 +123,112 @@ build, run the following inside vagrant:
 
   $ /project/sawtooth-core/bin/build_all
 
-Running validator
-=================
+Validator Start-up Process
+==========================
+
+Create Genesis Block
+--------------------
+
+In most use cases, it is not necessary to create a genesis block when starting
+a validator, because the validator joins an existing distributed ledger
+network. However, as a developer, you may often need to create short-lived
+test networks. In this case, you need to create a genesis block when
+instantiating a new network.
+
+The genesis block contains some initial values that are necessary when a
+Sawtooth Lake distributed ledger is created and used for the first time.
 
-To start a validator, log in to the development environment with 'vagrant ssh'
+To create the genesis block, log in to the development environment with the
+command `vagrant ssh` and run the following command:
+
+.. code-block:: console
+
+  $ sawtooth admin genesis
+  Generating /home/ubuntu/sawtooth/data/genesis.batch
+
+
+.. note:: 
+
+  If you need to delete previously existing block-chain data before running a
+  validator in the vagrant environment, simply run the following command: 
+  `rm /home/ubuntu/sawtooth/data/*`
+
+
+Start Validator
+---------------
+
+To start a validator, log in to the development environment with `vagrant ssh`
 and run the following commands:
 
 .. code-block:: console
 
    $ cd sawtooth-core/
-   $ validator -v
+   $ validator -vv
```

Comment:
@cintel version of generating a key works, but "sawtooth admin keygen" does not. 
Regardless, fixed and tested.